.eslintrc.yml

```diff
@@ -3,11 +3,16 @@ extends:
   - prettier
 ignorePatterns:
   - dist
+plugins:
+  - 'jest-formatting'
 rules:
   '@typescript-eslint/strict-boolean-expressions': off
   '@typescript-eslint/restrict-template-expressions': off
   '@typescript-eslint/naming-convention': off
   '@typescript-eslint/consistent-type-definitions': off
+  '@typescript-eslint/require-array-sort-compare': off
+  'jest-formatting/padding-around-describe-blocks': warn
+  'jest-formatting/padding-around-test-blocks': warn
```

Comment:
Popped these in in warn mode so they're won't poop the pipeline. It's an autofixable issue but I didn't wanna roll it out across all the test files and muddy this PR